<launch>
    <node pkg="hypergrid" type="pcl_to_gridmap" name="pcl_to_gridmap" output="screen">
        <rosparam command="load" file="$(find hypergrid)/config/pcl_to_gridmap.yaml"/>
        <param name="map_frame_id" value="base_footprint" />
        <!--remap from="velodyne_points" to="/$(arg vehicle)/velodyne_points" /-->
    </node>
</launch>